src/aap_eda/services/activation/activation_manager.py

```diff
@@ -203,7 +203,9 @@ def _start_activation_instance(self):
         self._create_activation_instance()
 
         self.db_instance.refresh_from_db()
-        log_handler = self.container_logger_class(self.latest_instance.id)
+        log_handler = self.container_logger_class(
+            self.latest_instance.id, LOGGER
```

Comment:
Either, with a logger instance or a logger name, now we have two arguments for the constructor and one of them is the same in all the calls, only `activation_instance_id` is dynamic. There is no need no repeat the same argument in all the calls, you can be more concise with a factory
```
def get_log_handler(self, activation_instance_id, logger=LOGGER):
    return self.container_logger_class(activation_instance_id, LOGGER)
```
So the lines like:
```
log_handler = self.container_logger_class(self.latest_instance.id)
```
become
```
log_handler = self.get_log_handler(self.latest_instance.id)
```